scalyr_agent/tests/scalyr_monitor_test.py

```diff
@@ -18,24 +18,31 @@
 __author__ = 'czerwin@scalyr.com'
 
 
+from scalyr_agent.json_lib import serialize
+from scalyr_agent.json_lib import parse as json_lib_parse
+from scalyr_agent.json_lib.objects import JsonArray, JsonObject, ArrayOfStrings
 from scalyr_agent.scalyr_monitor import MonitorConfig, BadMonitorConfiguration, define_config_option
 from scalyr_agent.test_base import ScalyrTestCase
 
 
 class MonitorConfigTest(ScalyrTestCase):
 
     def test_base(self):
+        test_array = ["a", 1, False]
+        test_obj = {"a": 100, "b": 200}
```

Comment:
I have augmented this test class to reflect additional support for converting between types.
The main type we need to support is `ArrayOfStrings`.  This gives us capability to support comma-separated list of strings in environment variable such as for `event_object_filter`.